src/test/java/org/openhab/ui/habot/test/TrainerNlTest.java

```diff
@@ -0,0 +1,292 @@
+        assertIsGetStatus ("gang lichten", "lichten", "gang");
+        assertIsGetStatus ("woonkamer verlichting", "verlichting", "woonkamer");
+        assertIsGetStatus ("keuken verlichting", "verlichting", "keuken");	
+        assertIsGetStatus ("welke lampen staan aan?", "lampen", null);
+        assertIsGetStatus ("wat is de temperatuur in de keuken?", "temperatuur", "keuken");
+        assertIsGetStatus ("staat de voordeur open?", "voordeur", null);
+        assertIsGetStatus ("is de voordeur open? ", "voordeur", null);
+        assertIsGetStatus ("welke ramen staan open?", "ramen", null);
+        assertIsGetStatus ("staat het raam in de wc open?", "raam", "wc");
+        assertIsGetStatus ("verwarming in de keuken", "verwarming", "keuken");
+        assertIsGetStatus ("Wat is de temperatuur in de woonkamer?", "temperatuur", "woonkamer");
+        assertIsGetStatus ("Wat is de temperatuur in de kelder?", "temperatuur", "kelder");
+    }
+	
+    @Test
+    public void testActivateObjects() throws Exception {
+        
+    	this.trainer = new IntentTrainer("nl", skills);
+        
+        assertIsActivate("zet de tv aan", "tv", null);
+        assertIsActivate("doe het licht aan", "licht", null);
+        assertIsActivate("doe het licht aan in de keuken", "licht", "keuken");
+        assertIsActivate("doe het licht aan op zolder", "licht", "zolder");
+        assertIsActivate("zet het licht aan op zolder", "licht", "zolder");
+        assertIsActivate("zet het licht aan in de gang", "licht", "gang");
+        assertIsActivate("zet het licht aan op de zolder", "licht", "zolder");
+        assertIsActivate("zet het licht op de gang aan", "licht", "gang");
+        assertIsActivate("zet het verwarmng aan", "verwarmng", null);
+        
+    }
+    
+    @Test
+    public void testDeactivateObjects() throws Exception {
+        
+    	this.trainer = new IntentTrainer("nl", skills);
+
+        assertIsDeactivate("doe het licht uit", "licht", null);
+        assertIsDeactivate("zet de tv uit", "tv", null);
+        assertIsDeactivate("doe het licht uit, alsjeblieft", "licht", null);
+        assertIsDeactivate("doe het licht uit", "licht", null);
+        assertIsDeactivate("doe het licht uit in de keuken", "licht", "keuken");
+        assertIsDeactivate("doe het licht uit op zolder", "licht", "zolder");
+        assertIsDeactivate("zet het licht uit op zolder", "licht", "zolder");
+        assertIsDeactivate("zet het licht uit in de gang", "licht", "gang");
+        assertIsDeactivate("zet het licht uit op de zolder", "licht", "zolder");
+        assertIsDeactivate("zet het licht op de gang uit", "licht", "gang");
+        assertIsDeactivate("zet het verwarmng uit", "verwarmng", null);
+    }
+    
+
+    @Test
+    public void testHistoryHourly() throws Exception {
+
+        Intent actual;
+        this.trainer = new IntentTrainer("nl", skills, null, "alphanumeric");
```

Comment:
I see some tests are using the "alphanumeric" tokenizer and others the default ("whitespace"). Please be aware is it set globally so it can only be one of them!